eng/performance/benchmark_jobs.yml

```diff
@@ -106,7 +106,7 @@ jobs:
             ${{ each framework in parameters.frameworks }}:
               ${{ framework }}:
                 _Framework: ${{ framework }}
-                _Configs: CompilationMode=Tiered
+                _Configs: CompilationMode=Tiered;RunKind="${{ parameters.kind }}"
```

Comment:
ci-setup.py's --build-configs option takes a space separated list. My suggestion would be to make each of these options space separated and put the quotes around the entire key=value, not around just the value. I.e. change it to

```
_Configs: "CompilationMode=Tiered" "RunKind=${{parameters.kind}}" 
```

But we should also check the logs to make sure that python isn't doing wonky things with the quotation marks.

Reply:
we have code in another place that parses this as semi-colon delimited, though.

We should pick one and make everything align.

Reply:
(I vote for semicolons..)

Reply:
Configs is only used by ci_setup.py.

Reply:
Ahh. It's super confusing that ci_setup is what makes this the semicolon delimited string that gets parsed later. Why'd we do it that way instead of representing the string the same way everywhere?

Reply:
I honestly don't care which of these that we do, so for now I am just going to go with the spaces and quotes. It does what we want, and is probably the safest.

Reply:
I was just following the example of other arguments that have multiple inputs. I guess it didn't really matter.